CRM/Chreports/Upgrader.php

```diff
@@ -1121,4 +1121,49 @@ public function upgrade_102009() {
     return TRUE;
   }
 
+  public function upgrade_102010() {
```

Comment:
This will be part of a regular release, not a hotfix... so version should be 1.3, not  1.2.3. Please update function accordingly.

```suggestion
  public function upgrade_103001() {
```